modules/proj4/src/lib/proj4-projection.js

```diff
@@ -13,6 +13,12 @@ export class Proj4Projection {
 
   constructor({from = 'WGS84', to = 'WGS84'}) {
     this.projection = proj4(from, to);
+    if (!this.projection) {
+      throw new Error('Undefined alias');
```

Comment:
It is not necessarily an alias, right?

Reply:
I suppose you're right... You might get an undefined result for an invalid projection string